### PR TITLE
Commits

### PR DESCRIPTION
**Pull request by AADI0009**
Fixed type in info command: lavalionk to lavalink
Added better values for vaporwave which sounds much better (i dont know if you would like it or not but give it a try, it sounds good)

### CODE REVIEW

File: commands/effect.go [hunk 1]
```diff
@@ -35,15 +35,9 @@ var effects = map[EffectType]lavalink.Filters{
...
-			0: 0.25,
-			1: 0.2,
-			2: 0.2,
-		},
 		Timescale: &lavalink.Timescale{
-			Pitch: 1,
-			Rate:  1,
-			Speed: 0.7,
+			Pitch: 0.8,
+			Speed: 0.80,
```
Comment (topi314) [inline commands/effect.go:40]: I have honestly no judgment if this sounds better, and tbh it doesn't really matter since this is a testbot anyway

Comment (AADI0009) [reply commands/effect.go:40]: True tho, i just shared some better values 👍🏻

**Comment by Drapersniper** [comment]
Keep in mind that "sounds much better" is subjective - both nightcore and vaporwave to be truly accurate need to be done on a per song basis.

**Comment by topi314** [comment]
> Keep in mind that "sounds much better" is subjective - both nightcore and vaporwave to be truly accurate need to be done on a per song basis.

based on this I'm gonna close this

**Pull request closed without merging**